tests/integration/requests_tests.py

```diff
@@ -149,6 +149,17 @@ def test_internal_client_sends_headers(http_server):
         assert http_server.requests[0].span.id != context.span.id
         assert http_server.requests[0].raw_edge_context == b"test payload"
 
+        setattr(context, "raw_edge_context", None)
```

Comment:
please split this into its own test so that it can pass/fail apart from the rest of the test.

Reply:
💇 Good call! Split it out as a separate case.